inst/rmarkdown/templates/pdf/resources/template.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[$if(papersize)$$papersize$paper,$else$letter,$endif$$if(fontsize)$$fontsize$,$else$9pt,$endif$$if(one_column)$$if(lineno)$lineno,$endif$$else$twocolumn,$endif$$if(one_sided)$$else$twoside,$endif$printwatermark=$if(watermark)$$watermark$$else$false$endif$]{pinp}
+\documentclass[$if(papersize)$$papersize$paper,$endif$$if(fontsize)$$fontsize$,$else$9pt,$endif$$if(one_column)$$if(lineno)$lineno,$endif$$else$twocolumn,$endif$$if(one_sided)$$else$twoside,$endif$]{pinp}
```

Comment:
Perhaps switch `printwatermark` to `displaywatermark`?

Reply:
Not needed, though. watermarks can be deprecated.

Reply:
I played for `s/print/display/` for a good 30 minutes or so last eve.  But ... if you look `pnas-new.cls` it is no longer a document options.

Things are borked at CRAN so I value a quick release over a possibly marginal feature we can always reintroduce if we value it.